src/Take.Blip.Builder/FlowManager.cs

```diff
@@ -411,6 +401,29 @@ private async Task<State> ProcessOutputsAsync(LazyInput lazyInput, IContext cont
             return state;
         }
 
+        private async Task<bool> EvaluateConditionCollectionAsync(
```

Comment:
Rename to `EvaluateConditionsAsync`
```suggestion
         private async Task<bool> EvaluateConditionsAsync(
```